src/Castle.Windsor.Tests/Registration/AllTypesTestCase.cs

```diff
@@ -617,5 +617,15 @@ public void RegisterGenericTypes_MultipleBasedOnWithTheSameTypesTwice_SelectedAs
 
 			Assert.AreEqual(1, services.Length);
 		}
+
+		[Test]
+		public void RegisterGenericTypes_WithoutDescriptor()
+		{
+			var ex = Assert.Throws<ArgumentException>(() => {
+				Kernel.Register(Classes
+					.FromAssembly(GetCurrentAssembly())
+				);
+			});
+		}
```

Comment:
Do we need to assign `var ex` if we are not making any assertions on the message returned?